lib/DonorsList/DonorsContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'react-final-form-arrays';
+
+import {
+  Col,
+  Loading,
+  Row,
+} from '@folio/stripes/components';
+
+import DonorsList from './DonorsList';
+import { useFetchDonors } from './hooks';
+
+function DonorsContainer({ name, donorOrganizationIds }) {
+  const [donorIds, setDonorIds] = useState(() => donorOrganizationIds);
+  const { donors, isLoading } = useFetchDonors(donorIds);
+
+  const handleFetchDonors = useCallback(ids => {
+    setDonorIds(ids);
+  }, []);
+
+  useEffect(() => {
+    if (donorOrganizationIds.length) {
+      handleFetchDonors(donorOrganizationIds);
```

Comment:
It seems you don't need extra function `handleFetchDonors` to update ids now

Reply:
updated

Reply:
Actually, I meant `handleFetchDonors` callback: it is redundant, because it does the same as a `setDonorIds`, so you can use it directly, and the `useEffect` was fine 🙂